### PR TITLE
PCA Notebook

### PR DESCRIPTION
**Pull request by sumedha-k**
@aarmey could you take a look at the PCA. The first two plots for from our last meeting, and the both bottom two are PCA for adcc and binding. We'll include the complement activation data once we get access to it.

### CODE REVIEW

**Comment by aarmey** [comment]
@sumedha-k this is exactly the right idea. You can move this into a figure file, and note that there are additional receptors with binding data.

**Comment by sumedha-k** [comment]
@aarmey I moved the PCA and R2X plots into figure6.py and ADCC cross validation stuff into figure7.py

File: deconv/figures/figure6.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import pandas as pd
+import numpy as np
+
+def makeFigure():
+    ax, f = getSetup((9, 3), (1, 3))
+
+    pca2 = PCA(n_components=5)
+
+    adcc3a, adcc3b = load_figures()
+    bindingA = pd.read_csv("./deconv/data/Fig2A-FcgRI.csv")
```
Comment (aarmey) [inline deconv/figures/figure6.py:14]: This all should be imported within a reusable function.

File: deconv/figures/oldinferfigure.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from .common import subplotLabel, getSetup
+from ..imports import load_tables, load_figures, infer_x_fixed, ADCC_groups
+
+
+def makeFigure():
```
Comment (aarmey) [inline deconv/figures/oldinferfigure.py:6]: If this is old you can get rid of it.

**Review by aarmey — CHANGES_REQUESTED** [review]
Great. Just two requests.

File: deconv/figures/figure6.py (hunk 1 above)
Comment (sumedha-k) [reply deconv/figures/figure6.py:14]: So should all of the data imports and pca computation be done in a separate function? Could you specify exactly which lines should be moved into a separate method? Should I put that method into imports.py?

File: deconv/figures/oldinferfigure.py (hunk 2 above)
Comment (sumedha-k) [reply deconv/figures/oldinferfigure.py:6]: ok, I will delete this file.

File: deconv/figures/figure6.py (hunk 1 above)
Comment (aarmey) [reply deconv/figures/figure6.py:14]: I'm just thinking all the `read_csv` lines. Other figures will need to do this, too.

**Pull request merged**